docs/api/en/math/Ray.html

```diff
@@ -118,7 +118,7 @@ <h3>[method:Boolean equals]( [param:Ray ray] )</h3>
 		<p>
 		[page:Ray ray] - the [page:Ray] to compare to.<br /><br />
 
-		Returns true if this and the other [page:Ray ray] have equal [page:.offset offset]
+		Returns true if this and the other [page:Ray ray] have equal [page:.origin origin]
```

Comment:
Can you please also fix this error in the Chinese version of the page 😇 ?

https://threejs.org/docs/index.html#api/zh/math/Ray.equals